pkg/test/resourcefixture/testdata/basic/secretmanager/v1beta1/secretmanagersecretversion/_generated_object_secretmanagersecretversion.golden.yaml

```diff
@@ -3,17 +3,20 @@ kind: SecretManagerSecretVersion
 metadata:
   annotations:
     cnrm.cloud.google.com/management-conflict-prevention-policy: none
+    cnrm.cloud.google.com/observed-secret-versions: (removed)
     cnrm.cloud.google.com/project-id: ${projectId}
+    cnrm.cloud.google.com/state-into-spec: absent
   finalizers:
   - cnrm.cloud.google.com/finalizer
   - cnrm.cloud.google.com/deletion-defender
-  generation: 2
+  generation: 3
   labels:
     cnrm-test: "true"
   name: secretmanagersecretversion-${uniqueId}
   namespace: ${uniqueId}
 spec:
   enabled: false
+  resourceID: "1"
```

Comment:
secret version is a service-genereated-ID resource, that TF_based always write back to `spec.resourceID`. in direct, we use `status.externalRef` instead.